README.md

```diff
@@ -29,26 +29,26 @@ To build an RPM package, you need to download this repository and use the `gener
     ```shellsession
     #./generate_rpm_package.sh -h
 
-    Usage: ./generate_rpm_package.sh [OPTIONS]
-
-        -b, --branch <branch>     [Required] Select Git branch or tag e.g.
-        -d, --destination <path>  [Required] Set the destination path of package.
-        -t, --target <target>     [Required] Target package to build [manager/api/agent].
-        -a, --architecture <arch> [Required] Target architecture of the package [x86_64/i386].
-        -r, --revision <rev>      [Required] Package revision that append to version e.g. x.x.x-rev
-        -l, --legacy              [Optional] Build package for CentOS 5.
-        -j, --jobs <number>       [Optional] Number of parallel jobs when compiling.
-        -p, --path <path>         [Optional] Installation path for the package. By default: /var.
+    Usage: ./generate_wazuh_rpm.sh [OPTIONS]
+
+        -b, --branch <branch>     [Required] Select Git branch [master]. By default: master.
+        -t, --target              [Required] Target package to build: manager, api or agent.
+        -a, --architecture        [Optional] Target architecture of the package. By default: x86_64
+        -j, --jobs                [Optional] Change number of parallel jobs when compiling the manager or agent. By default: 4.
+        -l, --legacy              [Optional] Build the package for CentOS 5.
+        -r, --release             [Optional] Package release. By default: 1.
+        -p, --path                [Optional] Installation path for the package. By default: /var.
+        -d, --debug               [Optional] Build the binaries with debug symbols. By default: no.
         -h, --help                Show this help.
     ```
     * To build a wazuh-manager package in /tmp for x86_64 and revision 3401:
-        `# ./generate_rpm_package.sh -b v3.4.0 -d /tmp -t manager -a x86_64 -r 3401`.
+        `# ./generate_rpm_package.sh -b v3.8.2 -d /tmp -t manager -a x86_64 -r 3401`.
     * To build a wazuh-agent package in /tmp for i386 with `-j15` and revision 3:
-        `# ./generate_rpm_package.sh -b v3.4.0 -d /tmp -t agent -a x86_64 -j 15 -r 3`.
+        `# ./generate_rpm_package.sh -b v3.8.2 -d /tmp -t agent -a x86_64 -j 15 -r 3`.
     * To build a wazuh-api package in /tmp from branch 3.6:
```

Comment:
If you say "To build a wazuh-api package in /tmp from branch 3.6" use branch 3.6 in the command. Change the branch or description.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/139/commits/eea92e2c82c08bcc398dee0c672de074c8323480